### ISSUE TITLE
long/lat mixup

### ISSUE THREAD

**Issue by ktk** [idris-maps/swiss-projection-light#1]
Hi,

thanks for the library, I'm using it in one of our projects. However I think there is a mixup, in the example you do:

    ch.lv03.toWgs.point([600000, 200000])

and in the documentation you say

    ch.<input-projection>.to<output-projection>.point([<x>,<y>], <round>)

But in the  Swiss LV03 coordinate system x is 2xx xxx and y is 6xx xxx. There is no explanation of what is sent back, I assumed it's lat and the long but in your example you create a POINT and if this is WKT then it's usually long/lat.

Long story short is I assumed I get back lat/long and that is clearly wrong. Also Swiss people would assume y is 6xx xxx and not 2xx xxx. Either way there is a bug in your interpretation or a lack of documentation or both :) Would be great if you could clarify that.

**Comment by ktk** [comment]
To be a bit more specific: in the API docs of Swisstopo I see y, x as argument and you get back lat/long.
